docs/csharp/misc/cs1751.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
@BillWagner Is this the correct directory for documenting new compiler diagnostics?

Reply:
You are right, I missed the folder. Changing it now. Thank you!